src/mach/templates/partials/endpoints/aws_endpoints.tf

```diff
@@ -0,0 +1,171 @@
+# API Gateway
+resource "aws_apigatewayv2_api" "{{ endpoint.key|slugify }}_gateway" {
+  name                       = "{{ site.identifier }}-{{ endpoint.key|slugify }}-api"
+  protocol_type              = "HTTP"
+}
+
+resource "aws_apigatewayv2_route" "{{ endpoint.key|slugify }}_application" {
+  api_id    = aws_apigatewayv2_api.{{ endpoint.key|slugify }}_gateway.id
+  route_key = "$default"
+}
+
+resource "aws_apigatewayv2_stage" "{{ endpoint.key|slugify }}_default" {
+  name                  = "$default"
+  description           = "Stage for default release"
+  api_id                = aws_apigatewayv2_api.{{ endpoint.key|slugify }}_gateway.id
+  auto_deploy           = true
+
+  {% if endpoint.throttling_burst_limit or endpoint.throttling_burst_limit -%}
+  default_route_settings {
+    {% if endpoint.throttling_burst_limit %}
+    throttling_burst_limit = {{ endpoint.throttling_burst_limit }}
+    {% endif %}
+    {% if endpoint.throttling_rate_limit %}
+    throttling_rate_limit = {{ endpoint.throttling_rate_limit }}
+    {% endif %}
+  }{% endif %}
+
+  depends_on = [
+    {% for component in endpoint.components %}
+    module.{{ component.name }},
+    {% endfor %}
+  ]
+}
+
+{% if endpoint.url %}
+  {% if endpoint.enable_cdn %}
+    provider "aws" {
```

Comment:
De provider wordt nu wel meerdere keren ge-init op het moment dat je meerdere endpoints met CDN hebt. Dat misschien ook bundelen (op de plek waar we alle providers aanmaken?)

Reply:
Fixed, maakt het idd overzichtelijker qua provider instanties